plugins/utils/ccc.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "ccc";
+  originalName = "ccc.nvim";
+  defaultPackage = pkgs.vimPlugins.ccc-nvim;
+
+  maintainers = [helpers.maintainers.JanKremer];
+
+  settingsOptions = {
+    default_color = helpers.defaultNullOpts.mkStr "#000000" ''
+      The default color used when a color cannot be picked. It must be HEX format.
+    '';
+
+    highlight_mode = helpers.defaultNullOpts.mkStr "bg" ''
+      Option to highlight text foreground or background. It is used to output_line and highlighter.
+      Options: `"fg" | "bg" | "foreground" | "background"`
```

Comment:
This could be an `enum` if those are the only valid values.

Also same comments regarding vim doc links (see below)